wordlists/names/top_male_name_germany.txt

```diff
@@ -0,0 +1,154 @@
+Aaron
+Abraham
+Adam
+Adalbert
+Adolf
+Aegidius
+Albert
+Alexander
+Alfred
+Alois
+Amand
+Andreas
+Anselm
+Anton
+August
+Augustin 
+Augustinus
+Balthasar
+Baptist
+Baruch
+Benedict
+Benjamin
+Bernhard
+Berthold/Barthold
+Bonaventur
+Bonifazius
+Carl
+Caspar
+Christian
+Christoph
+Christophorus
+Clemens
+Cleophas
+Cölestin
+Conrad
+Constantin
+Cyriakus
+Damasus
+Damian
+Daniel
+David
+Dionysius
+Dominik
+Dominikus
+Eberhard
+Eckhard
+Eduard
+Egyd
+Emanuel
+Emil
+Engelbert
+Engelhard
+Erhard
+Ernst
+Eugen
+Eustach
+Eustacius
+Ferdinand
+Fidel
+Florentius
+Franz
+Friedrich
+Fridolin
+Fritz
+Gambrich/ Gombrich
+Gabriel
+George
+Gerhard
+Germanus
+Gottlieb
+Gottfried
+Gregor
+Heinrich/Henrich,
+Henrikus
+Hermann
+Herzel
+Hieronymus
+Hirsch
+Ildefons
+Isaac
+Ignatz
+Ignatius
+Jacob
+Jean
+Johann
+Johannes
+Jonas
+Joseph
+Jost
+Josua
+Julius
+Justin
+Justus
+Karl
+Kasper
+Kilian
+Konrad
+Landelin
+Leopold
+Leonhard
```

Comment:
That's not alphabetically ordered.

Reply:
done the ordering

Reply:
Make sure everything is sorted with the website I've linked, I've just marked this one but there may be more.

Reply:
yeeah sure, i will check